research/src/pages/badge.proposal.mdx

```diff
@@ -0,0 +1,63 @@
+---
+menu: Proposals
+name: Badge (Editor's Draft)
+path: /components/badge
+pathToResearch: /components/badge.research
+---
+
+import BadgeAnatomy from '../components/badge-anatomy'
+
+# Badge Component Specification
+
+## Overview <a href="#overview" id="overview"></a>
+
+_Badge_ is normally a label holding small amount of information such as value or status. Appears as some sort of notification with eye-catching appeal.
+
+### Use Cases <a href="#use-cases" id="use-cases"></a>
+
+The <badge /> control is primarily leveraged to indicate value of notifications or status. For example a badge in a avatar can indicate status as busy, available and other while a badge in a inbox can indicate the number of new messages.
+
+## Prior Art/Examples <a href="#prior-art" id="prior-art"></a>
+
+- [Ant Design](https://ant.design/components/badge/)
+- [Bootstrap](https://getbootstrap.com/docs/4.3/components/badge/)
+- [Evergreen](https://evergreen.segment.com/components/badge-and-pill/)
+- [Material](https://material-ui.com/components/badges/#badge)
+- [FAST DNA](https://explore.fast.design/components/fast-badge)
+- [Atlas Kit](https://atlassian.design/components/badge/examples)
+- [Lightning](https://www.lightningdesignsystem.com/components/badges/)
+
+## &lt;badge&gt; Properties <a href="#badge-properties" id="badge-properties"></a>
+
+| Attribute Name | Type      | Default Value | Description                                                 |     |
+| -------------- | --------- | ------------- | ----------------------------------------------------------- | --- |
+| `value`        | `string`  | `""`          | Text value to be displayed by badge                         |     |
+| `circular`     | `boolean` | `true`        | If the badge shape is circular                              |     |
+| `color`        | `string`  | `red`         | Badge background color                                      |     |
+| `empty`        | `boolean` | `false`       | Badge can be rendered as a dot                              |     |
```

Comment:
Why would we need an `empty` state? could we simple have `value`, and hook into if `value` is empty?

Reply:
It can be a good strategy